src/Service/QueryParametersParser.php

```diff
@@ -30,6 +31,7 @@ public function __construct(FilterFactory $filterFactory)
         $this->filterFactory = $filterFactory;
 
         $this->parsers = new PrioritizedCollection(ParserInterface::class);
+        $this->parsers->add(new TupleColumnArrayValueParser($filterFactory), 4);
```

Comment:
Why this exact priority? What does it mean?
https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544

Also - if changed to eg. -333, the tests still pass.

Reply:
This PR is one of many - there is 6 parses ATM and I will send them one by one - their order is strict, so where there will be all of them it will makes more sense.

Reply:
Btw test passes even if you remove the line - since there is still the old implementation which fallbacks all yet not implemented parsers..

Reply:
Maybe, but does the order mean anything? Why it should be like this, if it won't break any tests?

Reply:
Because there are not all of them yet and it will break tests then because some parses may consume same _input_

Reply:
I've added the explicit priorities here https://github.com/lmc-eu/api-filter/pull/36